skema/text_reading/text_reading/webapp/public/schema/api-v1.yaml

```diff
@@ -7,6 +7,8 @@ info:
     name: Clayton T. Morrison
     email: claytonm@arizona.edu
 servers:
+  - url: https://skema.clulab.org
```

Comment:
Same as above. The use case will involve that our partners will run this through a docker compose file. Hopefully this is easy to support.

Reply:
> Same as above. The use case will involve that our partners will run this through a docker compose file. Hopefully this is easy to support.

This might be harder to change on the fly.  Hopefully there's a way to specify the current host or get the environment variable into the yaml.

Reply:
Okay, I'm sure we will find a way to deal with this